app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentViewModel.kt

```diff
@@ -954,6 +959,27 @@ class AdyenPaymentViewModel @Inject constructor(
     )
   }
 
+  fun processPayPalResult(paymentData: Observable<AdyenComponentResponseModel>) {
+    if (isFirstResultRun) {
```

Comment:
this seems a strange behavior. maybe the problem lies on another place for adding this logic...